### ISSUE TITLE
Artificially delay addition of report reaction on chatvotes

### ISSUE THREAD

**Issue by MoonlightCapital** [Dougley/MBv2#86]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when people instantly press the report reaction to a post as soon as it appears. I can't expect five seconds are enough to fully read and understand a suggestion if it's very long.

**Describe the solution you'd like**
Like the dupe confirmation reactions, the report in chatvotes should be artificially delayed. I suggest five seconds. Also, make it so the reaction appears only in custodian private channels, just to remove confusion in public ones.

**Additional context**
Maybe consider basing the delay on the length of the post?

**Comment by Dougley** [comment]
I disagree with this one, posting the link or running !info would imply that they've read the suggestion fully and deemed them invalid, and there's no positive benefit from just mindlessly hitting the reaction (reports need to be approved anyway)

**Comment by MoonlightCapital** [comment]
But shouldn't the same work for dupe, that you need to be sure of what you're doing before actually sending the command?

**Comment by Dougley** [comment]
The delay for dupe exists because dupe is more of a powertool than regular reporting, it has significantly more effect on the site than just deleting a single suggestion.

**Comment by jedi3r4** [comment]
I feel like even though the person who may be dropping the link may have fully read it, but others may not have. There should be a delay for report as well, so people can fully read it, instead of just react when it hits chat as well. 🤔

**Comment by Dougley** [comment]
https://github.com/Dougley/MBv2/commit/ceca2770f6d59134fe2207fec031de778f7ab6ea might've made this redundant?

**Comment by MoonlightCapital** [comment]
> [ceca277](https://github.com/Dougley/MBv2/commit/ceca2770f6d59134fe2207fec031de778f7ab6ea) might've made this redundant?

Not really, as it still takes time to read a post.
